docs/Documentation/Development/Jupyter/index.md

```diff
@@ -46,122 +52,171 @@ axes[3].set_title("fill_between");
+
+### Kestrel:
+
+`[user@laptop:~]$ ssh kestrel.hpc.nrel.gov`
+
+`[user@kl1:~]$ salloc -A <account> -t 02:00:00`
 
+### Eagle:
 
-#### ...Is more than one node possible?
+`[user@laptop:~]$ ssh eagle.hpc.nrel.gov`
 
-Yes... please see our advanced Jupyter documentation. 
+`[user@el1:~]$ salloc -A <account> -t 02:00:00`
 
-## Using a Compute Node - Hard Way
 
-### Start on a login node:
+## Starting Jupyter Inside the Job
 
-`ssh eagle.hpc.nrel.gov`
+Once the job starts and you are allocated a compute node, load the appropriate modules, activate your Jupyter environment, and launch the Jupyter server.
 
-`[user@el1:]$ srun -A <account> -t 02:00:00 --pty /bin/bash`
+#### Kestrel:
 
+`[user@x1000c0s0b0n1:~]$ module load anaconda3`
 
+`[user@x1000c0s0b0n1:~]$ source activate myjupenv`
 
+`[user@x1000c0s0b0n1:~]$ jupyter-notebook --no-browser --ip=$(hostname -s)`
 
+Take note of the node name that your job is assigned. (x1000c0s0b0n1 in the above example.)
 
-### When the job starts on the compute node:
+Also note the url that Jupyter displays when starting up, e.g. `http://127.0.0.1:8888/?token=<alphabet soup>`.
 
-`[user@r2i7n35]$ module load conda`
+The `<alphabet soup>` is a long string of letters and numbers. This is a unique authorization token for your Jupyter session. you will need it, along with the full URL, for a later step.
+
+#### Eagle:
+
+`[user@r2i7n35:~]$ module load conda`
 
 `source activate myjupenv`
 
 `jupyter-notebook --no-browser --ip=$(hostname -s)`
 
-note the node name (r2i7n35 in this example)
+Take note of the node name that your job is assigned. (r2i7n35 in this example.)
 
-and the url, e.g. `http://127.0.0.1:8888/?token=<alphabet soup>`
+Also note the url that Jupyter displays when starting up, e.g. `http://127.0.0.1:8888/?token=<alphabet soup>`.
 
-### In a terminal on your computer:
+The `<alphabet soup>` is a long string of letters and numbers. This is a unique authorization token for your Jupyter session. you will need it, along with the full URL, for a later step.
 
-`[user@laptop]$ ssh -N -L 8888:<nodename>:8888 username@eagle.hpc.nrel.gov`
-    
-copy full url from jupyter startup into your web browser. e.g.: 
+### On Your Own Computer
+
+Next, open an SSH tunnel through a login node to the compute node. Log in when prompted using your regular HPC credentials, and put this terminal to the side or minimize it, but leave it open until you are done working with Jupyter for this session.
+
+#### Kestrel: 
+
+`[user@laptop:~]$ ssh -N -L 8888:<nodename>:8888 username@eagle.hpc.nrel.gov`
+
+
+#### Eagle:
+
+`[user@laptop:~]$ ssh -N -L 8888:<nodename>:8888 username@eagle.hpc.nrel.gov`
+
+
+### Open a Web Browser
+
+Copy the full url and token from Jupyter startup into your web browser. For example:
 
 `http://127.0.0.1:8888/?token=<alphabet soup>`
 
 
-## Using a Compute Node - Easy Way
+## Using a Compute Node on Eagle - Easy Way
```

Comment:
The pyeagle package also works on Kestrel. Since the scripts we provide could also be pretty easily adapted to Kestrel by users too, maybe the section title should be changed to be non system specific like "Automate launching Jupyter on a compute node" or something similar.

Reply:
Removed the "on eagle" part again; section title could be further adapted if desired.